src/domUpdates.js

```diff
@@ -161,7 +161,7 @@ function displayIngredientsNeeded(recipe) {
         <p>${matchedId.name}: ${ingredient.amount} ${ingWithUnits.quantity.unit}</p><br>
       `
       return acc;
-    }, '');
+    }, '<h4>missing ingredients</h4>');
```

Comment:
Our reset file messed this up a bit so we needed to add this and the 146 p tag margin agove.